src/auto-evo/RunResults.cs

```diff
@@ -222,7 +222,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 else
                 {
                     builder.Append("  ");
```

Comment:
This should have one space added, right to reflect that " went extinct in" started with a space.